ios/FluentUI.Demo/FluentUI.Demo/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.3.19</string>
+	<string>1.4.0</string>
```

Comment:
@harrieshin I think updating the demo app to have the minor and revision numbers matching the library might be a handy to keep the correlation between app and library.
Thoughts?

Reply:
I mean I love it at least! Shouldn't be a problem until/if we hit 1.0 for Fluent.